src/wdio/services/TerraCommands/validate-element.js

```diff
@@ -21,8 +21,30 @@ const validateElement = (...args) => {
     misMatchTolerance,
   } = determineOptions.screenshotOptions(args);
 
-  accessibilityMethods.accessibleItBlock({ rules });
-  visualRegressionMethods.screenshotItBlock(name, selector, { misMatchTolerance });
+  global.it(`[${name}] is accessible and is within the mismatch tolerance`, () => {
+    accessibilityMethods.runAccessibilityTest({ rules });
+    visualRegressionMethods.runMatchScreenshotTest(selector, { misMatchTolerance, name });
+  });
 };
 
-export default validateElement;
+const validatesElement = (...args) => {
```

Comment:
you could DRY up these methods like you did for the accessibility functions above